qiskit/circuit/library/boolean_logic.py

```diff
@@ -21,41 +21,46 @@
 import numpy as np
 from qiskit.circuit import QuantumRegister, QuantumCircuit
 from qiskit.circuit.exceptions import CircuitError
+from qiskit.util import deprecate_arguments
 
 
 class Permutation(QuantumCircuit):
     """An n_qubit circuit that permutes qubits."""
 
+    @deprecate_arguments({'n_qubits': 'num_qubits'})
```

Comment:
I believe this hasn't been released yet so you don't have to deprecate the arg we can just change it up until the release